src/server/internalServer.cpp

```diff
@@ -455,6 +453,20 @@ std::unique_ptr<Response> InternalServer::handle_suggest(const RequestContext& r
     return Response::build_404(*this, request, bookName, "");
   }
 
+  auto start = 0;
+  try {
+    start = request.get_argument<unsigned int>("start");
+  } catch (const std::exception&) {}
+
+  unsigned int suggestionLength = 10;
+  try {
+    suggestionLength = request.get_argument<unsigned int>("suggestionLength");
```

Comment:
Just a small new remark.
`suggestionLength` seems a bit too long and error prone (different cases).
`count` is probably better.

Reply:
done!